acs-category-theory-notes.tex

```diff
@@ -1,28 +1,61 @@
-\documentclass[a4paper, 12pt]{article}
+%%%%%%%%%%%%%%%%%%%%%%%% NOTES TO CONTRIBUTORS %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% - try to stick to a minimal package set unless you need them. In other words,
+% don't use packages you don't understand simply because you're used to using
+% them.
+% - $$ eqn $$ is less universal than \[ eqn \] or \begin{equation} eqn \end{eqn}
+% - Don't use the math environment just to center things. That's what the center
+% environment is for.
+% - If you have other stylistic guidelines you want other poeple to keep in
+% mind, put them here
+
+
+% In the future we may want to break this into many subfiles (see the subfiles
+% package) so that we're not all working on a single monolithic document that
+% takes a long time to compile.
+
+\documentclass[a4paper, 12pt, oneside]{book}
+
 \usepackage{a4wide}
 \usepackage{nth}
-\usepackage{fourier} % Better font
-\usepackage{inconsolata} % mono spaced font
+% The "better font" is not scalable, also less standard
+%\usepackage{fourier} % Better font
+%\usepackage{inconsolata} % mono spaced font
```

Comment:
Admittedly we haven't used any monospaced text so far, and probably wont ever, so yes, this isn't needed. In fact I might go as far as to suggest that this line is deleted entirely - don't commit commented code.